Packs/AttachmentsPreProcessing/Scripts/GetFilePathPreProcessing/GetFilePathPreProcessing.py

```diff
@@ -0,0 +1,17 @@
+import time
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+attachments = demisto.incidents()[0].get("attachment")
+files = []
+if attachments is not None:
+    for attachment in attachments:
+        files.append({"path": attachment["path"], "name": attachment["name"]})
+
+    # The logic for finding the incident that we want to drop the new files to should be here
+    incident = "123"
+    demisto.executeCommand("executeCommandAt", {"command": "CreateFileFromPathObject",
+                           "arguments": {"object": files}, "incidents": incident})
+    time.sleep(10)
```

Comment:
There is no logic here to use- you can either add this code to the README file as example or add an actual logic rule for users to adopt.
In other cases, users can take the first script and wrap it with their own logic.